data/coalition/coalition missions.txt

```diff
@@ -91,12 +91,15 @@ mission "Discovered Coalition Space"
 		fail
 
 event "label coalition space"
-	galaxy "label saryds"
-		sprite "label/saryds"
-	galaxy "label kimek"
-		sprite "label/kimek"
-	galaxy "label arachi"
-		sprite "label/arachi"
+	galaxy "Milky Way"
+		label "saryds"
+			sprite "label/saryds"
+	galaxy "Milky Way"
+		label "kimek"
+			sprite "label/kimek"
+	galaxy "Milky Way"
```

Comment:
Can't you be able to just add all three labels at once? If not, then you should be capable of doing that; we don't have to remention the fleet we're adding a variant to for each new variant added by an event, for example.
```suggestion
		label "kimek"
			sprite "label/kimek"
```